### PR TITLE
create reusable buttons

### PR DESCRIPTION
**Pull request by prokivskaaa**

### CODE REVIEW

**Comment by prokivskaaa** [comment]
https://fe-feb23-code-hunters.github.io/fe_phone-catalog/

File: src/components/shared/buttons/MoveButton/MoveButton.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  isDisabled,
+  iconClassName,
+  onClick,
+}) => {
+  const ArrowIcon = isDisabled ? Arrow : ArrowDark;
+
+  return (
+    <IconButton isDisabled={isDisabled} onClick={onClick}>
+      <ArrowIcon className={iconClassName} />
+    </IconButton>
```
Comment (Hamuud) [inline src/components/shared/buttons/MoveButton/MoveButton.tsx:21]: If we see the design sigma, something going wrong, when icon is disabled: 
image

And Figma design: 
image

**Review by Hamuud — CHANGES_REQUESTED** [review]
Everything else is good, I don't even know what's wrong, except for this button, maybe I'm wrong, and it should look like this

File: src/components/shared/buttons/MoveButton/MoveButton.tsx (hunk 1 above)
Comment (prokivskaaa) [reply src/components/shared/buttons/MoveButton/MoveButton.tsx:21]: @Hamuud Check one more time please

**Review by Hamuud — APPROVED** [review]

**Pull request merged**